README.md

```diff
@@ -39,8 +39,16 @@ ln -fs openshift-ansible-*<version>* openshift-ansible
 
 | openshift-ansible url     | OpenShift version | 
 |:-------------------------:|:-----------------:|
+| [3.5.77-1](https://github.com/openshift/openshift-ansible/archive/openshift-ansible-3.5.77-1.tar.gz) | OCP 3.5.x |
 | [3.4.50-1](https://github.com/openshift/openshift-ansible/archive/openshift-ansible-3.4.60-1.tar.gz) | OCP 3.4 |
 
+* Run `ansible-galaxy` to pull in the necssary requirements for the CASL provisioning:
+
+```
+cd ~/src/casl-ansible
+ansible-galaxy install -r casl-requirements.yml -p roles
+```
+
 Cool! Now you're ready to provision OpenShift clusters
 
 ### Provision a Cluster
```

Comment:
@etsauer not sure I quite follow - it seems that you may have it backwards. I.e.: if you choose to run a "normal" provisioning job that builds a cluster with DNS, it will **indeed** exercise the role(s) that were moved over. In either case, it will use one or both of the roles currently moved. Anyway; maybe this can be part of the bigger revamp that needs to be happen for this README to restructure / clarify the usage?